config/manager/deployment.yaml

```diff
@@ -31,6 +31,12 @@ spec:
         securityContext:
           allowPrivilegeEscalation: false
           readOnlyRootFilesystem: true
+          capabilities:
+            drop: [ "ALL" ]
+          runAsUser: 65534
+          runAsGroup: 65534
+          seccompProfile:
+            type: RuntimeDefault
```

Comment:
This syntax requires Kubernetes `1.19`. I am assuming this is OK as it aligns with `flux check --pre` which checks for `>=1.19.0`.

Reply:
We do support older version, see. https://fluxcd.io/docs/installation/#prerequisites